# goal
string package_name
geometry_msgs/Pose package_pose
---
# result
bool success
---
# feedback
geometry_msgs/Pose arm_pose